jdaviz/core/template_mixin.py

```diff
@@ -4358,9 +4479,16 @@ def __init__(self, plugin, viewer_type='scatter', app=None, *args, **kwargs):
                     ]
         self._initialize_toolbar()
 
+        plugin.session.hub.broadcast(PluginPlotAddedMessage(sender=self))
+        plugin.session.hub.broadcast(PluginPlotModifiedMessage(sender=self))
```

Comment:
Is there a reason to broadcast both here?

```suggestion
```

Reply:
i cant remember why i did this, but there was a reason at the time haha. ill remove and test without.

Reply:
If it turns out to be necessary, please just add a comment so we don't forget and accidentally remove it in the future

Reply:
i tested and i don't think its necessary so i removed the line